### PR TITLE
Update private.py - quantity / notional precision

### PR DESCRIPTION
**Pull request by JuanitoLC**
Hello. I'm using your code to access to Crypto Exchange. While testing my code i realize that when i was trying to buy CRO with BTC, there was en error.
In function create_order, you were modifying 'quantity' always with the quantity precision, so when I wanted to sell BTC to buy CRO, the quantity passed was 0.00 and there was an exception.
In fact, when we sell BTC to buy CRO, we have to use 'notional', so i changed the way you modify the precision with price precision. Now it works.

### CODE REVIEW

File: src/cryptocom/exchange/private.py [hunk 1]
```diff
@@ -166,10 +166,12 @@ async def create_order(
         data['time_in_force'] = force_type.value
         data['exec_inst'] = exec_type.value
 
-        quantity = "{:.{}f}".format(quantity, pair.quantity_precision)
+        # here is the change i propose
```
Comment (mortyspace) [inline src/cryptocom/exchange/private.py:169]: Nice! Look you can do this not duplicating formating string:
```
quantity_precision = pair.quantity_precision
if ...
    quantity_precision = pair.price_precision
quantity = "{:.{}f}".format(quantity, quantity_precision )
data['quantity'] = quantity
```

Remove comment "#", add version 0.10.1 into README and change __version__ variable and I will accept pull and will release new version)

Comment (JuanitoLC) [reply src/cryptocom/exchange/private.py]: I removed the comment.

Comment (JuanitoLC) [reply src/cryptocom/exchange/private.py]: and added version 0.10.1 into README and changed version __inti__.py

sorry, I'm new in github and i don't know if i did correctly.

**Comment by mortyspace** [comment]
can you update from master pls to fix tests, ty

**Comment by Merlin153** [comment]
Hello Heer is Hakan Yerdekalmaz
I want chatt with you
My whats App number 
004916099565062

**Comment by Merlin153** [comment]
Thank welcome
When you work by bank send a credit card
With your name and id number
Is cool what you done

**Comment by Merlin153** [comment]
Stader street 95
28205 east Bremen

**Pull request closed without merging**